clusters/app.ci/supplemental-ci-images/telco-runner.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-runner
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-runner
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: telco-runner:latest
+    source:
+      dockerfile: |
+        FROM registry.ci.openshift.org/openshift/centos:stream9
+        ENV GOCACHE=/go/.cache \
+            GOARM=5 \
+            GOPATH=/go \
+            GOFLAGS='-mod=vendor' \
+            LOGNAME=deadbeef \
+            HOME=/output
+        RUN dnf install -y ansible-core wget make jq which && \
+            dnf clean all && \
+            wget https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.11.9/openshift-client-linux.tar.gz && \
+            tar -zxvf openshift-client-linux.tar.gz && \
+            mv oc /usr/local/bin/ && \
+            mv kubectl /usr/local/bin/ && \
+            rm -f openshift-client-linux.tar.gz && \
+            mkdir /output && \
+            chmod -R g+rwx /output && \
+            mkdir -p $GOPATH/bin && \
+            chmod g+xw -R $GOPATH && \
+            chmod g+w /etc/passwd && \
+            echo 'echo default:x:$(id -u):$(id -g):Default Application User:/output:/sbin/nologin\ >> /etc/passwd' > /output/fix_uid.sh && \
+            chmod g+rwx /output/fix_uid.sh
+        RUN for version in 1.19 1.20 1.21.6; do \
+            mkdir -p /usr/local/${version} && \
+            wget https://golang.org/dl/go${version}.linux-amd64.tar.gz && \
+            tar -C /usr/local/${version} -xzf go${version}.linux-amd64.tar.gz && \
+            chmod g+xw -R /usr/local/${version}/go && \
+            rm go${version}.linux-amd64.tar.gz && \
+            echo "export GOLANG_VERSION=${version}" > /output/golang-${version} && \
```

Comment:
isn't `/output` remounted in the job? If so, we'll lose everything in there, just checking.

Reply:
Hello @sshnaidm 

Apologies, I didn't fully understand what you meant by "remounted in the job."

I'm following the pattern of the existing "/output/fix_uid.sh" file. We create this file in the "/output" directory, which is the home directory because of the "HOME=/output" environment variable. 

~~~
echo 'echo default:x:$(id -u):$(id -g):Default Application User:/output:/sbin/nologin' >> /etc/passwd' > /output/fix_uid.sh && \

~~~

Currently, During our step execution we have fix_uid.sh file in the home dir so I following this pattern I think "golang-${version}" should also be available.

https://github.com/openshift/release/blob/master/ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh#L8-L9

~~~
~/fix_uid.sh
~~~